setup.py

```diff
@@ -62,21 +62,12 @@ def run(self):
     license='MIT License',
     packages=['pew'],
     install_requires=[
-        'virtualenv>=1.11', 'virtualenv-clone>=0.2.5', 'setuptools>=17.1'
+        'virtualenv<20.0.0,>=1.11', 'virtualenv-clone>=0.2.5', 'setuptools>=17.1'
```

Comment:
I use virtualenv 20.0 with pew regularly. It works fine for the features I use, and is *far* faster than the older version. So I'm a strong -1 on forcing an older version here.

If there are pew features that are not available with the new virtualenv version, they should fail individually.